### PR TITLE
Add gas-part smoothing-length histogram

### PR DESCRIPTION
**Pull request by EvgeniiChaikin**
Add gas-particle smoothing-length histogram to the colibre pipeline

**Example**

Screenshot from 2020-12-12 17-22-31

### CODE REVIEW

**Comment by MatthieuSchaller** [comment]
I have copied over the script to the XL pipeline as well. All looks good to me. Thanks!

File: eagle-xl/scripts/gas_smoothing_lengths.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+Makes a histogram of the gas smoothing lengths. Uses the swiftsimio library.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from swiftsimio import load
+from unyt import unyt_quantity, dimensionless, Mpc
+
+h_bounds = [1e-2, 5e2]  # comoving kpc
```
Comment (JBorrow) [inline eagle-xl/scripts/gas_smoothing_lengths.py:11]: @MatthieuSchaller are we sure we want the same bounds for XL?

Comment (MatthieuSchaller) [reply eagle-xl/scripts/gas_smoothing_lengths.py]: We use 500kpc as our max h so technically no need to go bigger. I have nevertheless increased the range a bit so the the max is seen.

**Comment by MatthieuSchaller** [comment]
@JBorrow any final words?

**Pull request merged**